src/analysis/analytic/inverse.lean

```diff
@@ -478,7 +478,7 @@ begin
     have B : ∀ᶠ a in 𝓝 0, r * (I + 1) * a < 1/2,
       by { apply (tendsto_order.1 this).2, simp [zero_lt_one] },
     have C : ∀ᶠ a in 𝓝[>] (0 : ℝ), (0 : ℝ) < a,
-      by { filter_upwards [self_mem_nhds_within], exact λ a ha, ha },
+      by { filter_upwards [self_mem_nhds_within] using λ _ ha, ha },
```

Comment:
```suggestion
      by { filter_upwards [self_mem_nhds_within] with _ ha using ha },
```
It's a bit longer but I think we should encourage always using the `with` form when we intend to follow up with a lambda.